mods/ts/fluent/rules.ftl

```diff
@@ -732,15 +732,15 @@ actor-stnk =
     .description =
     Lightly armored tank equipped with a personal
     stealth generator. Armed with missiles.
-    Visible to infantry at close range.
+    Can be spotted by infantry at close range.
        Strong vs Vehicles and Aircraft.
        Weak vs Infantry.
 
 actor-sgen =
     .name = Mobile Stealth Generator
     .deployed-name = Mobile Stealth Generator (deployed)
     .description =
-    Able to cloak units once deployed.
+    Can cloak units when deployed.
```

Comment:
Maybe this needs a better rewording. In game it cloaks all allied forces in an area around it